tests/secc/states/test_messages.py

```diff
@@ -201,6 +203,34 @@ def get_dummy_v2g_message_power_delivery_req_charge_start():
     )
 
 
+def get_v2g_message_power_delivery_req_charge_start_invalid_charging_profile():
+    charging_profile = ChargingProfile(
```

Comment:
It would be good to add two tests that covers the above mentioned use case as well - which is return a charging profile that doesn't fall on the same boundary as the chosen schedule - but is in line with the limits and one that is not.

Reply:
done